src/cs/ui/InfoBar.cs

```diff
@@ -98,4 +104,20 @@ public partial class InfoBar : ProgressBar {
 	public void _HideInfo() {
 		Box.Hide();
 	}
+	
+	// The bar can have 2 colors, one when it's low and one normal
+	public void _UpdateColor(bool l) {
+		if (l) {
+			StyleBoxFlat Stylebox = this.GetThemeStylebox("fill").Duplicate() as StyleBoxFlat;
```

Comment:
Using `this` is only useful in the case of naming collisions, it should be omitted otherwise.